core/src/main/java/org/restcomm/media/core/configuration/MgcpControllerConfiguration.java

```diff
@@ -35,17 +35,17 @@
     
     public static final String ADDRESS = "127.0.0.1";
     public static final int PORT = 2427;
-    public static final int MGCP_BUFFER_SIZE = 5000;
+    public static final int CHANNEL_BUFFER = 5000;
 
     private String address;
     private int port;
-    private int mgcpBufferSize = MGCP_BUFFER_SIZE;
+    private int channelBuffer = CHANNEL_BUFFER;
```

Comment:
already initialised in default constructor.

Reply:
Done